At the time of the integration of pytz 2013d, Solaris ships version 2012j
of the Olson database.  While there are other changes in the database, a
few zoneinfo files have been introduced since the version we ship, so we
remove those from the hard-coded list in pytz.

--- pytz-2013d/pytz/__init__.py	Tue Oct 22 14:37:28 2013
+++ pytz-2013d/pytz/__init__.py	Tue Oct 22 14:39:58 2013
@@ -762,7 +762,6 @@
  'Asia/Kashgar',
  'Asia/Kathmandu',
  'Asia/Katmandu',
- 'Asia/Khandyga',
  'Asia/Kolkata',
  'Asia/Krasnoyarsk',
  'Asia/Kuala_Lumpur',
@@ -804,7 +803,6 @@
  'Asia/Ulaanbaatar',
  'Asia/Ulan_Bator',
  'Asia/Urumqi',
- 'Asia/Ust-Nera',
  'Asia/Vientiane',
  'Asia/Vladivostok',
  'Asia/Yakutsk',
@@ -913,7 +911,6 @@
  'Europe/Brussels',
  'Europe/Bucharest',
  'Europe/Budapest',
- 'Europe/Busingen',
  'Europe/Chisinau',
  'Europe/Copenhagen',
  'Europe/Dublin',
@@ -1044,7 +1041,6 @@
  'Pacific/Yap',
  'Poland',
  'Portugal',
- 'ROC',
  'ROK',
  'Singapore',
  'Turkey',
@@ -1321,7 +1317,6 @@
  'Asia/Karachi',
  'Asia/Kashgar',
  'Asia/Kathmandu',
- 'Asia/Khandyga',
  'Asia/Kolkata',
  'Asia/Krasnoyarsk',
  'Asia/Kuala_Lumpur',
@@ -1357,7 +1352,6 @@
  'Asia/Tokyo',
  'Asia/Ulaanbaatar',
  'Asia/Urumqi',
- 'Asia/Ust-Nera',
  'Asia/Vientiane',
  'Asia/Vladivostok',
  'Asia/Yakutsk',
@@ -1400,7 +1394,6 @@
  'Europe/Brussels',
  'Europe/Bucharest',
  'Europe/Budapest',
- 'Europe/Busingen',
  'Europe/Chisinau',
  'Europe/Copenhagen',
  'Europe/Dublin',
